extensions/amp-browsi/validator-amp-browsi.protoascii

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-browsi
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-browsi"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "pub-id"
+}
+tags: {  # <amp-browsi>
+  html_format: AMP
+  tag_name: "AMP-BROWSI"
+  requires_extension: "amp-browsi"
+  attr_lists: "pub-id"
```

Comment:
I don't see any attribute lists defined as `pub-id`. I think you want to allow `pub-key` and `site-key` and maybe put some restrictions on their values. It may be helpful if you look at [Contributing Validator Rules for an AMP Extended Component](https://github.com/ampproject/amphtml/blob/master/contributing/component-validator-rules.md).

Reply:
Fixed it, thanks.